_components/TableOfContents.css

```diff
@@ -9,7 +9,7 @@
 @media (min-width: 1024px) {
   .content {
     display: grid;
-    grid-template-columns: minmax(auto, 960px) 1fr;
```

Comment:
This bit was the problematic code; `auto` allows the element's ancestors to dictate width. Setting `0` as the minimum overrides this and forces the box to just take up the column given. (8:3 felt like the right ratio to me, but it could be a different ratio.)